app/gui2/src/components/GraphEditor/widgets/WidgetNumber.vue

```diff
@@ -19,6 +19,15 @@ const value = computed({
     })
   },
 })
+
+const limits = computed(() => {
+  const config = props.input.dynamicConfig
+  if (config?.kind === 'Numeric_Input' && config?.minimum != null && config?.maximum != null) {
```

Comment:
The other ? should not be necessary anymore.
```suggestion
  if (config?.kind === 'Numeric_Input' && config.minimum != null && config.maximum != null) {
```